.idea/libraries/Maven__junit_junit_4_10.xml

```diff
@@ -0,0 +1,13 @@
+<component name="libraryTable">
+  <library name="Maven: junit:junit:4.10">
+    <CLASSES>
```

Comment:
don't include any files under .idea by using a .gitignore file. 
review last week's chat if you don't know how to use a .gitignore file.